app/views/layouts/application.html.haml

```diff
@@ -7,13 +7,11 @@
     = csp_meta_tag
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+
   %body
-    = render partial: 'partials/header'
-    #content
-      - flash.each do |key, value|
-        = value
-      - if current_page?(root_path)
-        = render partial: 'partials/carousell'
-      = yield
-      
-      
+    - @campaigns.each do |campaign|
+      %h1= campaign.title
+      %p= campaign.description
+      %p= campaign.location
+
+  = yield
```

Comment:
I asked you to comment out code. Not remove it.

Reply:
Also, there is a whole lot of classes you can use to style this. Just look at the original site.